cli/internal/cmd/info/bin.go

```diff
@@ -14,7 +16,7 @@ func BinCmd(helper *cmdutil.Helper) *cobra.Command {
 		Use:   "bin",
 		Short: "Get the path to the Turbo binary",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			base, err := helper.GetCmdBase(cmd.Flags())
+			base, err := helper.GetCmdBase(config.FlagSet{FlagSet: cmd.Flags()})
```

Comment:
what does `FlagSet` do?

Reply:
It's a wrapper for `pflag.FlagSet` that we need to implement the `CLIConfigProvider` interface. Turns out Go doesn't let you implement interfaces for non-local types (even stricter than Rust!)